### PR TITLE
Add Spanish translation

### PR DESCRIPTION
**Pull request by oscfdezdz**

### CODE REVIEW

File: res/com.github.alexhuntley.Plots.metainfo.xml [hunk 1]
```diff
@@ -5,10 +5,12 @@
   <metadata_license>FSFAP</metadata_license>
   <project_license>GPL-3.0-or-later</project_license>
   <name>Plots</name>
+  <name xml:lang="es">Plots</name>
   <name xml:lang="fr">Plots</name>
   <name xml:lang="hr">Grafikoni</name>
   <developer_name>Alex Huntley</developer_name>
   <summary>Simple graph plotting</summary>
+  <summary xml:lang="es">Trazado de funciones sencillas</summary>
```
Comment (alexhuntley) [inline res/com.github.alexhuntley.Plots.metainfo.xml:13]: I don't know much Spanish, but shouldn't it be "sencillo" (singular)? It is the app which should be simple, not (necessarily) the functions being plotted.

**Review by alexhuntley — COMMENTED** [review]
Looks good to me, just had one suggestion about the wording of the summary (found in both files, I've only commented on one of them).

File: res/com.github.alexhuntley.Plots.metainfo.xml (hunk 1 above)
Comment (oscfdezdz) [reply res/com.github.alexhuntley.Plots.metainfo.xml]: It should in that case 😅 It's fixed now.

**Comment by alexhuntley** [comment]
Great, thanks for the contribution!

**Pull request merged**